robotpy.feature/feature.xml

```diff
@@ -104,7 +104,7 @@ This Agreement is governed by the laws of the State of New York and the intellec
    </license>
 
    <requires>
-      <import feature="org.python.pydev.feature" version="3.9.1" match="greaterOrEqual"/>
+      <import feature="org.python.pydev.feature" version="6.0.0.201709191431" match="greaterOrEqual"/>
```

Comment:
This could probably be greater or equal to 6.0.0

Reply:
Ill change that